package.json

```diff
@@ -13,10 +13,11 @@
     "build:illustrations": "node ./cli/empty-state-illustration-generator.js",
     "build:sass": "node-sass --output-style expanded ./test/sass.scss > ./test/sass.output.css",
     "build": "npm run build:clean && npm run build:icons && npm run build:illustrations && npm run build:sass",
+    "build-static": "rollup -c",
     "lint": "npm run lint:eslint && npm run lint:style && npm run lint:lit",
-    "lint:eslint": "eslint . --ext .js,.html",
+    "lint:eslint": "eslint . --ext .js,.html --ignore-pattern build",
```

Comment:
Would the `--ignore-path .gitignore` flag make sense here?

Reply:
Sounds good to me! And yeah it works on my end... the extra things it'll exclude are:

- `generated/` (only relevant if you lint after building...)
- `test/sass.output.css` (same as above)

Now, in our current pipelines we do in fact lint after building, so I guess the above are relevant.